src/lib/Commands/TestCommand/index.js

```diff
@@ -337,10 +388,18 @@ ${bootstrapCode}
 }
 
 // resolve modules
-__module_tests(__module_ImpUnit_Promise_exports, __module_impUnit_exports.ImpTestCase);
```

Comment:
The Promise is in impUnit\index.nut  ->  #require "promise.class.nut:3.0.0" so the Promise 3.0.0 is present accessible in code as Promise and as ::Promise. Promise, JSONEncoder and Predix tests have been executed.